demo/app.js

```
@@ -0,0 +1,2120 @@
+
+function columnLetter(index) {
+  let letter = '';
+  while (index >= 0) {
+    letter = String.fromCharCode(65 + (index % 26)) + letter;
+    index = Math.floor(index / 26) - 1;
+  }
+  return letter;
+}
+
+function escapeHtml(str) {
+  return str.replace(/&/g, '&amp;')
```

Biome (2.1.2)

[error] Shouldn't redeclare 'escapeHtml'. Consider to delete it or rename it.

'escapeHtml' is defined here:


(lint/suspicious/noRedeclare)